settings.py

```diff
@@ -565,6 +565,8 @@
 FORMPLAYER_URL = 'http://localhost:8080'
 OFFLINE_TOUCHFORMS_PORT = 4444
 
+USE_FORMPLAYER = False
```

Comment:
why can't we use the toggle you setup here? https://github.com/dimagi/commcare-hq/pull/9999/files#diff-57f91d9d96727cedf8225cb4c7d50b74R633